### PR TITLE
librsvg: update to 2.52.6

### PR DESCRIPTION
**Pull request by mmuetzel**
Also fix building with clang.

### CODE REVIEW

File: mingw-w64-librsvg/PKGBUILD [hunk 1]
```diff
@@ -57,11 +57,11 @@ build() {
     --enable-vala \
     --enable-silent-rules
 
-  make
+  make RUSTFLAGS="-C linker=${CC}"
```
Comment (lazka) [inline mingw-w64-librsvg/PKGBUILD:60]:
```suggestion
  make
```

File: mingw-w64-librsvg/PKGBUILD [hunk 2]
```diff
@@ -57,11 +57,11 @@ build() {
...
 }
 
 package() {
   cd "${srcdir}/build-${MSYSTEM}"
-  make DESTDIR="${pkgdir}" install
+  make install DESTDIR="${pkgdir}" RUSTFLAGS="-C linker=${CC}"
```
Comment (lazka) [inline mingw-w64-librsvg/PKGBUILD:65]:
```suggestion
  make DESTDIR="${pkgdir}" install
```

File: mingw-w64-librsvg/PKGBUILD (hunk 1 above)
Comment (mmuetzel) [reply mingw-w64-librsvg/PKGBUILD]: Done

**Comment by lazka** [comment]
Builds fine locally -> ship it

**Comment by dustwanderer** [comment]
I am not sure whether I have understood you. I tried executing doublecmd.exe from a bash shell, or from Windows PowerShell. The error message did not appear in either case. However, If I execute doublecmd.exe from "Windows Command Prompt (CMD.exe)", or from "Windows File Explorer", I get the error message

**Comment by Biswa96** [comment]
I have tried that also and doublecmd.exe executes without any issue. Here is the commands:

```cmd
set PATH=C:\msys64\ucrt64\bin;%PATH%
doublecmd.exe
```

**Comment by dustwanderer** [comment]
Your commands provided the solution. I moved C:\msys64\ucrt64\bin to the top of my PATH stack, and the error disappeared.

It turns out that my PATH had one version of librsvg-2-2.dll, but two versions of libxml2-2.dll. So, I was loading an obsolete libxml2-2.dll instead of the updated libxml2-2.dll in C:\msys64\ucrt64\bin\.

I removed the obsolete libxml2-2.dll, and rearranged my PATH to its original configuration.

Thank you very much for your help.

**Pull request merged**